packages/realm/src/binding.ts

```diff
@@ -16,7 +16,27 @@
 //
 ////////////////////////////////////////////////////////////////////////////
 
-import { IndexSet, Int64, ObjKey, SyncSession, Timestamp, WeakSyncSession } from "realm/binding";
+import {
+  GeoBox as ExternalGeoBox,
+  GeoCircle as ExternalGeoCircle,
+  GeoPoint as ExternalGeoPoint,
+  GeoPolygon as ExternalGeoPolygon,
+  IGeoPoint,
+  IGeoPosition,
+} from "./internal";
```

Comment:
We can't have this import from `internal`. At the very least, make this an `import type` so that it has no runtime impact. But really this feels like the wrong place for this logic. At least in my opinion, this file should only depend on the binding and add some simpler helpers that are easiest to implement in JS/TS and don't depend on the rest of the SDK. For stuff like this, I think we want the normal `toBinding`-style functions that live in the SDK layer rather than in the binding layer.

Reply:
I think it makes sense. I'll move this to `GeoSpatial.ts`